services_app/src/androidTest/java/org/opendatakit/utilities/AbstractPermissionsTestCase.java

```diff
@@ -1427,12 +1427,9 @@ private boolean ableToChangeRow(String rowId, boolean isTableLocked, boolean pri
       }
       // or we are the owner
       // TODO: perhaps more logic here if test coverage is more complete
```

Comment:
Same complaint here. The comments are pointing to concepts under test and comments aren't updated and is probably a change that makes a person understand the code less.

Reply:
I have made the changes and pushed. Thank you